<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	border:3px solid red;
	border-radius:10px;
	background-color:black;
	color:black;
}
QPushButton{
	border-color:black;
}
QPushButton#Btn_min:hover{	
	background-color: rgba(226, 226, 226, 200);
}
QPushButton#Btn_close:hover{
	background-color:rgb(255,84,57);
}
QToolButton{
	border-radius:10px;
	background-color:white;
}
QToolButton:hover{
     border-radius:10px;
     font-size:17px;
     background-color:pink;
     color:yellow;
}
QLineEdit{
	border:2px;
	border-radius:10px;
	border-color:rgb(255,255,191);
	background-color:white;
}
QLineEdit:hover{
	border:2px solid pink;
	border-radius:10px;
}</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtn_Backspace">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Backspace</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtn_Clear">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tBtn_Clear_all">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_min">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>0</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="Btn_close">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>0</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="tBtn_MC">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>MC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_7">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_div">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_sqrt">
      <property name="text">
       <string>Sqrt</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QToolButton" name="tBtn_MR">
      <property name="text">
       <string>MC</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_mul">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_pow">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QToolButton" name="tBtn_MS">
      <property name="text">
       <string>MS</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_sub">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_pow2">
      <property name="text">
       <string>1/x</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QToolButton" name="tBtn_Madd">
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_point">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_changes">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_add">
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="tBtn_equal">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Btn_min</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>11</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
